config.yaml

```
@@ -22,4 +23,10 @@ steps:
       snapshot_bucket: "open-targets-pre-data-releases"
       snapshot_base_path: "snapshots"
       opensearch_java_opts: "-Xms2g -Xmx4g"
-  
\ No newline at end of file
+  ot_croissant:
+    - name: ot_croissant generate release metadata
+      ftp_address: http://ftp.ebi.ac.uk/pub/databases/opentargets/platform/24.09/output/etl/parquet/
+      gcp_address: gs://open-targets-data-releases/24.09/output/etl/parquet/
+      date_published: "2025-02-12"
+      dataset_path: /home/ricardo/Documents/work-files/etl/testfiles/outputs/parquet/targets
```

```suggestion
      dataset_path: output/target
```

Why only `target`? Is this as a test? If it is for testing purposes for now, it is fine.

Although this should work like it is, the path should be relative to the `work_path` in line 2 to avoid having to hardcode local paths in the repo.